src/main/java/homework4/Bank.java

```diff
@@ -0,0 +1,40 @@
+package homework4;
+
+import java.util.HashMap;
+
+public class Bank<Integer, V extends Asset> extends HashMap<Integer, V> {
```

Comment:
What reason og generic Integer? You can create some parent of pank !=Integer?

Reply:
Sorry, bad look

Reply:
because Bank expect only integer in account id.